src/sentry/sentry_apps/api/endpoints/sentry_app_details.py

```diff
@@ -17,11 +17,14 @@
     catch_raised_errors,
 )
 from sentry.api.serializers import serialize
-from sentry.api.serializers.rest_framework import SentryAppSerializer
 from sentry.auth.staff import is_active_staff
 from sentry.constants import SentryAppStatus
 from sentry.mediators.sentry_app_installations.installation_notifier import InstallationNotifier
 from sentry.organizations.services.organization import organization_service
+from sentry.sentry_apps.api.serializers.parsers.sentry_app import RequestSentryAppSerializer
```

Comment:
SentryAppParser would be a nice classname to have. Would mirror the Serializer conventions without overlapping terms.